stripe/api_resources/charge.py

```diff
@@ -23,6 +28,54 @@ class Charge(
     """
 
     OBJECT_NAME = "charge"
+    amount: int
+    amount_captured: int
+    amount_refunded: int
+    application: Optional[Any]
+    application_fee: Optional[Any]
+    application_fee_amount: Optional[int]
+    authorization_code: str
+    balance_transaction: Optional[Any]
+    billing_details: StripeObject
+    calculated_statement_descriptor: Optional[str]
+    captured: bool
+    created: str
+    currency: str
+    customer: Optional[Any]
+    description: Optional[str]
+    disputed: bool
+    failure_balance_transaction: Optional[Any]
+    failure_code: Optional[str]
+    failure_message: Optional[str]
+    fraud_details: Optional[StripeObject]
+    id: str
+    invoice: Optional[Any]
+    level3: StripeObject
+    livemode: bool
+    metadata: Dict[str, str]
+    object: Literal["charge"]
+    on_behalf_of: Optional[Any]
+    outcome: Optional[StripeObject]
+    paid: bool
+    payment_intent: Optional[Any]
+    payment_method: Optional[str]
+    payment_method_details: Optional[StripeObject]
+    radar_options: StripeObject
+    receipt_email: Optional[str]
+    receipt_number: Optional[str]
+    receipt_url: Optional[str]
+    refunded: bool
+    refunds: Optional[Any]
+    review: Optional[Any]
+    shipping: Optional[StripeObject]
+    source: Optional[Any]
+    source_transfer: Optional[Any]
+    statement_descriptor: Optional[str]
+    statement_descriptor_suffix: Optional[str]
+    status: str
+    transfer: Any
```

Comment:
```suggestion
    transfer: Union[str, StripeObject]
```

?

Reply:
Out of scope? I will do expandables and union types separately so it easier to focus on the logic specific to the more involved cases.